lib/octokit/client/apps.rb

```diff
@@ -80,10 +82,12 @@ def create_integration_installation_access_token(installation, options = {})
       # @param options [Hash] A customizable set of options
       # @see https://developer.github.com/v3/apps/installations/#list-repositories
       #
-      # @return [Array<Sawyer::Resource>] A list of repositories
+      # @return [Sawyer::Resource] A list of repositories
```

Comment:
Likewise here, I think this actually returns a resource containing a `total_count` and a list of `repositories`.